.github/workflows/main.yml

```diff
@@ -45,3 +50,4 @@ jobs:
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+          platforms: 'linux/amd64,linux/arm64'
```

Comment:
This approach runs each build sequentially and can be a little slow as you start adding new platforms.

Would you be comfortable making a change to follow [this approach](https://docs.docker.com/build/ci/github-actions/multi-platform/#distribute-build-across-multiple-runners) instead? It allows for builds to run in parallel and will keep the time to publish pretty much unchanged, regardless of how many platforms you choose to build.

Reply:
Sure, let me update this PR

Reply:
We want to build docker images ourselves in github and then push to docker hub, because builds in docker hub are painfully slow. We'll hold off this change until then